tests/tests_default_reboot.yml

```diff
@@ -0,0 +1,10 @@
+- name: Ensure that the rule runs with default parameters
+  hosts: all
+  vars:
+    kdump_reboot_ok: true
+  tags:
+    - tests::reboot
+  tasks:
+    - name: Run the role and reboot if necessary
+      include_role:
+        name: linux-system-roles.kdump
```

Comment:
should check that `kdump_reboot_required` is `false`, and also use a `wait_for_connection` to ensure the node was rebooted correctly https://docs.ansible.com/ansible/2.9/modules/wait_for_connection_module.html#wait-for-connection-module

not sure how you could tell if the node was rebooted correctly - you would have to know ahead of time if the node needed to be rebooted *and* that the role correctly rebooted the node

Reply:
Done.
After the reboot, the role starts the kdump service. It then returns error if the service does not start correctly. Just like in the [tests_default_reboot-FAILED.log](https://dl.fedoraproject.org/pub/alt/linuxsystemroles/logs/lsr-citool_kdump-84-6b8b5b3_20211130-201520/artifacts/tests_default_reboot-FAILED.log) test for Fedora 35.

Reply:
so then, how to tell if the role correctly rebooted the node if a reboot was required?

Reply:
Do you mean correctly rebooted in regard to kdump or in total?

After the reboot, the role starts the kdump service. If the kdump service does not properly, the role fails like it does in BaseOS CI RHEL 6 default_reboot test. If the kdump service is started properly, the role passes successfully.